cmake/configs/base.cmake

```diff
@@ -307,6 +307,12 @@ config_bool(
     DEFAULT OFF
 )
 
+config_bool(
+    ENABLE_MODEL
```

Comment:
The idea behind the model is that it will eventually expand to cover enough WT to make RTS testing possible. In other words, the model will eventually model a non-trivial part of WT (including RTS), and if this approach becomes successful in helping with RTS, it will be expanded (in a separate project) to test other parts of WT. As such "ENABLE_KV_MODEL" is way too narrow—even as I will be soon expanding to support transactions and other things.

But if "ENABLE_MODEL" is way too non-specific, we can rename it to "ENABLE_WT_MODEL" for clarity. I just thought that the "WT" part was not necessary.

Reply:
Ack - got it, no problem with the existing naming then.